.github/workflows/new-version.yml

```diff
@@ -21,27 +21,27 @@ jobs:
           node-version: 14
           cache: 'yarn'
 
-      - name: Yarn install
+      - name: Yarn install Hero
         run: yarn && yarn build
         working-directory: ./hero
         env:
           HERO_SKIP_CHROME_DOWNLOAD: 1
           ULX_CHROMEALIVE_SKIP_BINARY_DOWNLOAD: 1
 
-      - name: Yarn install
+      - name: Yarn install Databox
         run: yarn && yarn build
         working-directory: ./databox
         env:
           HERO_SKIP_CHROME_DOWNLOAD: 1
           ULX_CHROMEALIVE_SKIP_BINARY_DOWNLOAD: 1
```

Comment:
Are these three lines needed for Databox?

Reply:
Databox has a hero-core dependency, so I thought it might and it's painful to check github changes... so I just put them in there in case